src/index.ts

```diff
@@ -1,42 +1,30 @@
 let cant = document.getElementById("cantidad");
 
-function iniciar_butacas(max: number): boolean[] {
+function iniciarButacasAleatoriamente(cantidad: number): boolean[] {
   let butacas: boolean[] = [];
-  for (let i = 0; i < max; i++) {
-    butacas[i] = false;
+  for (let i = 0; i < cantidad; i++) {
+    butacas[i] = Math.random() < 0.5;
   }
   console.log(butacas);
   return butacas;
 }
```

Comment:
- Cambiamos el nombre de la función a **iniciarButacasAleatoriamente**.
- Ese for solo pone valores **false** en el arreglo. Tenemos que generar valores **true** y **false** aleatoriamente, y eso se puede hacer `Math.random() < 0.5;`.